traffic_ops/traffic_ops_golang/role/roles.go

```diff
@@ -164,6 +164,11 @@ func (role *TORole) Read() ([]interface{}, error, error, int) {
 	if userErr != nil || sysErr != nil {
 		return nil, userErr, sysErr, errCode
 	}
+
+	if version.Major > 1 {
+		return vals, nil, nil, http.StatusOK
```

Comment:
this needs to fill in the capabilities else it will return nil for capabilities

Reply:
It that case I'll just move this into the loop below.
Done.